### PR TITLE
Add an `identify` command #FF

### PR DESCRIPTION
**Pull request by anandaroop**
Adds an `identify` command to identify an Artist, Artwork or Partner based on its BSON id. This also demonstrates the oclif way of mocking network requests in the accompanying test.

Screen Shot 2019-09-01 at 5 40 30 PM
 
Still supplying the token as an env var, till we get logins wired up.

### CODE REVIEW

**Comment by anandaroop** [comment]
Haha I think Circle is lying to us about all that 🍏

**Comment by anandaroop** [comment]
Got this working, once I figured out how to `nock` correctly.

File: test/commands/identify.test.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  describe("when the artwork exists", () => {
+    test
+      .nock("https://stagingapi.artsy.net", api =>
+        api
+          .get("/api/v1/artist/abc123")
+          .reply(404)
+          .get("/api/v1/artwork/abc123")
+          .reply(200)
+          .get("/api/v1/partner/abc123")
+          .reply(404)
```
Comment (anandaroop) [inline test/commands/identify.test.ts]: The second arg to `.reply` could be an object representing the JSON response.

File: src/commands/identify.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const { args } = this.parse(Identify)
+    const { id } = args
+
+    const gravityPromises = Identify.collectionsToCheck.map(collection => {
+      const resource = `${collection.endpoint}/${id}`
+      return gravity.get(resource)
+    })
+
+    const gravityResponses = await Promise.all(gravityPromises)
+    const foundIndex = gravityResponses.findIndex(r => r.status === 200)
```
Comment (anandaroop) [inline src/commands/identify.ts]: The strategy here is to build up all the requests for the various collections, fire em off all at once, and look for one that succeeded.

**Comment by jonallured** [comment]
Ok just had to update an assumption about the Gravity urls based on what @pepopowitz did over on #8.

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.4 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.4 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.4 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.4 :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in v0.0.4 :rocket:

**Pull request merged**